activemerchant.gemspec

```diff
@@ -28,6 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency("offsite_payments", "~> 2.0.0")
 
   s.add_development_dependency('rake')
+  s.add_development_dependency('test-unit', '~> 1.2.3')
```

Comment:
I added this as an explicit dependency. 
- 1.2.3 is outdated, but was the version that was bundled with Ruby 1.8
- Ruby 1.9 bundles Minitest instead, but includes a shim so Test::Unit tests still work.
- Ruby 2.2 drops the Test:Unit shim.

So, in order to reliably test against different Ruby versions, we have to explicitly specify either `test-unit` or `minitest`. Because we are using `Test::Unit::TestCase` subclasses for tests, and Minitest has version conflicts with different activesupport versions, I decided on using test-unit.